sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectQuery.cs

```diff
@@ -111,8 +111,13 @@ public override AdtQueryBuilder Build()
         /// <inheritdoc/>
         public override string GetQueryText()
         {
+            // handle passing in null
```

Comment:
I don't find "what this code is doing" kinds of comments helpful unless the what is unusual. I'm usually more interested in "why", and in this case, the why seems pretty obvious. We can't operate on `_clause.ClauseArgs` if it is null.